css/properties/glyph-orientation-vertical.json

```
@@ -24,7 +24,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤4"
```

It wasn't present in 3, but is present in 4, see [here](https://github.com/caugner/css-properties/commit/cf1b5de97bcb5f335533005f5fdd8ff327bcfc11#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328R127):

```suggestion
              "version_added": "4"
```